lib/widgets/relax_timer/play_pause_button.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:flutter/material.dart';
+
+class PlayPauseButton extends StatelessWidget {
+  final VoidCallback onPressed;
+  final Icon icon;
+  final Widget label;
+
+  PlayPauseButton({
+    required this.onPressed,
+    required this.icon,
+    required this.label,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return FloatingActionButton.extended(
+      onPressed: () async {
```

Comment:
you don't need braces here I think, only passing pointer to a function, something like `onPressed: onPressed`

Reply:
change successful.